lib/jekyll-github-metadata/ghp_metadata_generator.rb

```diff
@@ -9,11 +9,17 @@ class GHPMetadataGenerator < Jekyll::Generator
       attr_reader :site
 
       def generate(site)
+        if defined?(@_already_generated)
+          Jekyll::GitHubMetadata.log :debug, "Nothing left to do!"
```

Comment:
We already are exposing users to unnecessary friction with unclear output, I'd suggest making it clear what's going on here.